### PR TITLE
Added go-broadcaster

### PR DESCRIPTION
**Pull request by mikluke**

### CODE REVIEW

File: internal/blockchain/blockchain.go [hunk 1]
```diff
@@ -62,80 +46,8 @@ func (b *blockchain) DistributeReward(ctx context.Context, receiver sdk.AccAddre
...
-}
-
-func (b *blockchain) Ping(_ context.Context) error {
-	c, err := b.ctx.GetNode()
-	if err != nil {
-		return fmt.Errorf("failed to get rpc client: %w", err)
-	}
-	if _, err := c.Status(); err != nil {
-		return fmt.Errorf("failed to check node status: %w", err)
+	if err := b.b.BroadcastMsg(msg, ""); err != nil {
```
Comment (megaherz) [inline internal/blockchain/blockchain.go:49]: Why did you remove the retry?

Comment (mikluke) [reply internal/blockchain/blockchain.go:49]: It's inbuilt in broadcaster
